pkg/jira/edit.go

```diff
@@ -323,6 +323,8 @@ func constructCustomFieldsForEdit(fields map[string]string, data *editRequest) {
 			switch configured.Schema.DataType {
 			case customFieldFormatOption:
 				data.Update.M.customFields[configured.Key] = []customFieldTypeOptionSet{{Set: customFieldTypeOption{Value: val}}}
+			case customFieldFormatProject:
+				data.Update.M.customFields[configured.Key] = []customFieldTypeFormatSet{{Set: customFieldTypeFormat{Value: val}}}
```

Comment:
maybe this?

```suggestion
				data.Update.M.customFields[configured.Key] = []customFieldTypeProjectSet{{Set: customFieldTypeProject{Value: val}}}
```

Reply:
yes, a bit quick on the pushing there :)